docs/openstack/advanced-openstack-topics/setting-up-your-own-images/how-to-build-windows-image.md

```diff
@@ -33,204 +33,220 @@ upload the image to the NERC's OpenStack Compute cloud. Please follow the follow
 
-3. Customize the Virtual machine:
+![Virt-Manager Selected ISO](images/3.4.select_iso.png)
 
-    ![Virt-Manager Customize Image](images/7.0.customize_iso.png)
+![Virt-Manager default Memory and CPU](images/4.default_mem_cpu.png)
 
-    Enable the **VirtIO** driver. By default, the Windows installer does not
-    detect the disk.
+Please set **15 GB disk image** size as shown below:
 
-    ![Virt-Manager Disk with VirtIO driver](images/7.1.customize_sata_disk_virtio.png)
+![Virt-Manager disk image size](images/5.set_15_GB_disk_size.png)
 
-    Add a **New Hardware**  > select **CDROM device** and attach to downloaded
-    **virtio-win-* ISO** file:
+Set the virtual machine name and also make sure **"Customize configuration before
+install"** is selected as shown below:
 
-    ![Virt-Manager Add Hardware](images/7.4.add_virtio_iso_hardware.png)
+![Virt-Manager Virtual Machine Name](images/6.set_name.png)
 
-    ![Virt-Manager Add CDROM with virtio ISO](images/7.5.add_virtio_iso_cdrom.png)
+### 3. Customize the Virtual machine
 
-    ![Virt-Manager Browse virtio ISO](images/7.6.browse_virtio_iso.png)
+![Virt-Manager Customize Image](images/7.0.customize_iso.png)
 
-    ![Virt-Manager Select virtio ISO](images/7.7.select_virtion_iso.png)
+Enable the **VirtIO** driver. By default, the Windows installer does not
+detect the disk.
 
-    Make sure the NIC is using the **virtio** Device model as show below:
+![Virt-Manager Disk with VirtIO driver](images/7.1.customize_sata_disk_virtio.png)
 
-    ![Virt-Manager Modify  NIC](images/7.2.customize_nic_virtio.png)
+Add a **New Hardware**  > select **CDROM device** and attach to downloaded
+**virtio-win-* ISO** file:
 
-    ![Virt-Manager Apply Change on NIC](images/7.3.customize_nic_virtio_apply.png)
+![Virt-Manager Add Hardware](images/7.4.add_virtio_iso_hardware.png)
 
-    Make sure to set proper order of **Boot Options** as shown below so that
-    CDROM with Windows ISO is set on the first and Apply the order change.
-    After this please begin windows installation clicking on **"Begin Installation"**
-    button.
+![Virt-Manager Add CDROM with virtio ISO](images/7.5.add_virtio_iso_cdrom.png)
 
-    ![Windows Boot Options](images/7.8.boot_option_win_cdrom_first.png)
+![Virt-Manager Browse virtio ISO](images/7.6.browse_virtio_iso.png)
 
-4. Click "Apply" button and continue with the Windows installation.
+![Virt-Manager Select virtio ISO](images/7.7.select_virtion_iso.png)
 
-    When prompted you can choose "Windows Server 2022 Standard Evaluation
-    (Desktop Experinece)" option as shown below:
+Make sure the NIC is using the **virtio** Device model as show below:
 
-    ![Windows Desktop Installation](images/7.windows_installation_desktop.png)
+![Virt-Manager Modify  NIC](images/7.2.customize_nic_virtio.png)
 
-    ![Windows Custom Installation](images/8.custom_setup.png)
+![Virt-Manager Apply Change on NIC](images/7.3.customize_nic_virtio_apply.png)
```

Comment:
might be worth adding a step that states to click Apply?

Reply:
I added click "Apply" button before Step #4